.github/workflows/ci-tests/Examples_tests/test_launcher.sh

```diff
@@ -106,6 +147,27 @@ function flash_with_openocd() {
         openocd_dapLink_pid=$!
     fi
 }
+
+#****************************************************************************************************
+# params: target in lower case, DAP sn
+function reset_board_by_openocd() {
+    echo "function: ${FUNCNAME[0]} $@"
+
+    set +e
+    $OPENOCD -f $OPENOCD_TCL_PATH/interface/cmsis-dap.cfg \
+             -f $OPENOCD_TCL_PATH/target/$1.cfg -s $OPENOCD_TCL_PATH \
+             -c "adapter serial $2" \
+             -c "gdb_port 3333" -c "telnet_port 4444" -c "tcl_port 6666" \
+             -c "init; reset run" >/dev/null &
+
+    openocd_dapLink_pid=$!
+    sleep 0.5
+    if ps -p $openocd_dapLink_pid >/dev/null; then
+        kill -9 $openocd_dapLink_pid || true
+    fi
+    set -e
+}
+
```

Comment:
My understanding per the comment is that this function takes 2 arguments for example: 
reset_board_by_openocd some_device  some_device_serial
so $1 = some_device
and $2 = some_device_serial
but you are calling it with 3 arguments
reset_board_by_openocd max32655 MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID
reset_board_by_openocd $DUT_NAME_LOWER $DUT_NAME_LOWER $DUT_ID

so here $1= max32655 and  $2= max32655 and  $3= serial
so you are never seeing the serial in the function because it is argument 3. 
Should look like this:
reset_board_by_openocd $MAIN_DEVICE_NAME_LOWER $MAIN_DEVICE_ID
reset_board_by_openocd $DUT_NAME_LOWER $DUT_ID